trilium_alchemy/core/entity/model.py

```diff
@@ -78,31 +76,31 @@ class Model(ABC):
     """
 
     # pydantic model used in etapi
-    etapi_model: Type[BaseModel] = None
+    etapi_model: Type[BaseModel] | None = None

```

Comment:
These ones (through `fields_default`) should be overridden by subclasses, so should never be `None` - we can just remove the assignment